CHANGELOG.md

```diff
@@ -7,29 +7,9 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## 17.10.0
-
-* Enable data attributes for table header links (PR #962)
-
-## 17.9.0
-
-* Use shared_helper for heading margin (PR #958)
-
-## 17.8.0
-
-* Remove old cookie banner code and new_cookie_banner flag (PR #959)
-
-## 17.7.0
-
-* Add an inverse flag to the phase banner (PR #954)
-
-## 17.6.1
-
-* Fix confirmation message being read to screenreaders (PR #952)
-* Hide 'Accept Cookies' button when Javascript not available (PR #948)
-
-## 17.6.0
```

Comment:
Why has all of this been removed from the changelog? You shouldn't be removing anything, it's a record of what's changed.

Reply:
@JonathanHallam @andysellick I'm guessing this might have happened as a result of a bad rebase?

Reply:
Yeah, we're looking at it now.